.github/workflows/python-package.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master, EVA-3050 ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
Does Nextflow run without specifying a Java dependency here?

Reply:
It does apparently: https://github.com/apriltuesday/eva-assembly-ingestion/actions/runs/3410080493/jobs/5672606842

Maybe it defaults to the most recent version of Java on the runner?